OpenVote/src/com/openvote/GoodbyeServlet.java

```diff
@@ -0,0 +1,20 @@
+package com.openvote;
+
+
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GoodbyeServlet extends HttpServlet {
+	
+
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+    	
+    	// any clean-up required here?
+    	
+    	resp.sendRedirect("/goodbye.jsp");
```

Comment:
If this is all the servlet does, I don't think we need it. We can just have a simple redirect whenever we're going to goodbye